### ISSUE TITLE
Not all rows exported

### ISSUE THREAD

**Issue by t-schroeder** [mtxr/vscode-sqltools#717]
**Describe the bug**
Often when exporting the result of a query with more than 50 rows it will only export the first 50 rows.

Output -> SQLTools:

> [1603380055437] INFO  (ext): EXECUTING COMMAND => sqltools.saveResults
> [1603380062108] INFO  (ls): REQUEST RECEIVED => connection/SaveResultsRequest

**To Reproduce**
1. Create table with more than 50 rows.
2. Select * from table_name;
3. Click on 'export'
4. Store as csv.
5. Notice that only the first 50 rows got exported.

**Expected behavior**
All rows should get exported.

**Desktop (please complete the following information):**
 - SQLTools Version 0.23.0
 - VSCode Version: 1.50.1
 - OS: Windows 10 connected to Linux (CentOS 7) via Remote SSH extension
 - Driver: 
   - [ ] PostgreSQL/Redshift
   - [x] MySQL/MariaDB
   - [ ] MSSQL/Azure
   - [ ] SQLite
   - [ ] Other? Which...
 - Database version: mysql  Ver 15.1 Distrib 10.2.34-MariaDB, for Linux (x86_64) using readline 5.1

**Additional context**

**Comment by zmp2000** [comment]
Hi, I think it should have an option to configure the pagination size, or infinity .
Should I open a new issue with that suggestion?

**Comment by t-schroeder** [comment]
Good idea. I would consider that a separate issue though, since it would only be a workaround and not really solve the problem I described.
